### PR TITLE
add covalent translations

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

Original PR: https://github.com/moonbeam-foundation/moonbeam-docs/pull/717/

### Checklist

- [ ] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
     - [ ] Double-check images

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/integrations/indexers/covalent.md [hunk 1]
```diff
@@ -1,135 +1,263 @@
...
- - Covalent API的根URL为https://api.covalenthq.com/v1/
- - 所有请求均通过HTTPS完成（若使用HTTP则请求失败）
+ - The current version of the API is version 1
+ - 当前API版本为V1
+ - The default return format for all endpoints is JSON
+ - 所有端点的默认返回形式为JSON
+ - All requests require authentication; you will need [a free API Key](https://www.covalenthq.com/platform/#/auth/register/){target=_blank} to use the Covalent API
+ - 所有请求均要验证，用户需要[可用的API密钥](https://www.covalenthq.com/platform/#/auth/register/){target=_blank}才能使用Covalent API
+ - The cost of an API call is denominated in credits and varies depending on the particular call. Upon creating an API key, you're given a substantial amount of free credits to get started (100,000 at the time of writing). You can track your usage of these free credits on the [Increment Dashboard](https://www.covalenthq.com/platform/increment/#/?utm_source=moonbeam&utm_medium=partner-docs){target=_blank}
+ - API调用的成本以积分计价，并根据特定调用而有所不同。创建API密钥后，您将获得大量免费积分开始操作（截至本文撰写时积分为100,000）。您可以在[Increment数据面板](https://www.covalenthq.com/platform/increment/#/?utm_source=moonbeam&utm_medium=partner-docs){target=_blank}上追踪这些免费积分的使用情况
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:69]:
```suggestion
 - API调用的收费以credit计价，并根据特定调用而有所不同。创建API密钥后，您将获得大量免费credit开始操作（截至本文撰写时为100,000）。您可以在[Increment数据面板](https://www.covalenthq.com/platform/increment/#/?utm_source=moonbeam&utm_medium=partner-docs){target=_blank}上追踪这些免费credit的使用情况
```

File: builders/integrations/indexers/covalent.md [hunk 2]
```diff
@@ -1,135 +1,263 @@
...
+ - The root URL of the API is: [https://api.covalenthq.com/v1/](https://api.covalenthq.com/v1/){target=_blank}
+ - API的根URL为[https://api.covalenthq.com/v1/](https://api.covalenthq.com/v1/){target=_blank}
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:71]:
```suggestion
 - API的root URL为[https://api.covalenthq.com/v1/](https://api.covalenthq.com/v1/){target=_blank}
```

File: builders/integrations/indexers/covalent.md [hunk 3]
```diff
@@ -1,135 +1,263 @@
...
 
-## 支持终端 {: #supported-endpoints }
+### Types of Endpoints - 端点类型 {: #types-of-endpoints }
+
+The Covalent API has three classes of endpoints:
+
+Covalent API有三类端点：
+
+ - **Class A** — endpoints that return enriched blockchain data applicable to all blockchain networks, eg: balances, transactions, log events, etc
+ - **A类** — 可以在所有区块链网络中使用，能够返回详细区块链数据的端点，包括余额、交易和日志事件等
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:84]:
```suggestion
 - **A类** — A端点提供丰富的在所有区块链网络共通的数据，包括余额、交易和日志事件等
```

File: builders/integrations/indexers/covalent.md [hunk 4]
```diff
@@ -1,135 +1,263 @@
...
+ - **Class B** — endpoints that are for a specific protocol on a blockchain, e.g. Uniswap is Ethereum-only and is not applicable to other blockchain networks
+ - **B类** — 专门为特定区块链上的特定协议而设置的端点，例如Uniswap就是仅适用于以太坊的端点，无法在其他区块链网络上使用
+ - **Class C** — endpoints that are community built and maintained but powered by Covalent infrastructure.
+ - **C类** — 由社区构建并维护，但由Covalent基础设施提供支持的端点
+
+### Sample Supported Endpoints - 支持端点的示例 {: #sample-supported-endpoints }
+
+For a full list of supported endpoints, refer to the [Covalent API reference](https://www.covalenthq.com/docs/api/guide/overview/){target=_blank}. A subset of the supported endpoints include:
+
+支持端点的完整列表请查看[Covalent API参考文档](https://www.covalenthq.com/docs/api/guide/overview/){target=_blank}。支持端点的子集包括：
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:94]:
```suggestion
支持端点的完整列表请查看[Covalent API参考文档](https://www.covalenthq.com/docs/api/guide/overview/){target=_blank}。支持端点包括但不限于：
```

File: builders/integrations/indexers/covalent.md [hunk 5]
```diff
@@ -1,135 +1,263 @@
...
- - 具有拥有一定数量资金的账户。
- --8<-- 'text/_common/faucet/faucet-list-item.md'
+To get started with the Covalent API, you will need to have the following:
 
-## Covalent API使用测试 {: #try-it-out }
+要开始使用Covalent API，您需要准备以下内容：
 
-首先确保您已获得以“ckey_”开头的[API密钥](https://www.covalenthq.com/platform/#/auth/register/){target=_blank}。Token Holders终端将返回某一特定代币所有持有者的列表，调用这个API需要执行以下操作：
+ - A free [Covalent API Key](https://www.covalenthq.com/platform/#/auth/register/){target=_blank}
+ - 可用的[Covalent AP密钥](https://www.covalenthq.com/platform/#/auth/register/){target=_blank}
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:197]:
```suggestion
 - 免费的[Covalent AP密钥](https://www.covalenthq.com/platform/#/auth/register/){target=_blank}
```

File: builders/integrations/indexers/covalent.md [hunk 6]
```diff
@@ -1,135 +1,263 @@
...
- - 您的API密钥
- - Moonbase Alpha Chain ID: {{ networks.moonbase.chain_id }} (hex: {{ networks.moonbase.hex_chain_id }})
- - 智能合约（此示例中为ERTH代币）：0x08B40414525687731C23F430CEBb424b332b3d35
+### Using Curl - 使用Curl {: #using-curl }
 
-### 使用Curl  {: #using-curl }
+One of the supported endpoints is the token holders endpoint, which returns a list of all the token holders of a particular token. For this example, you can check the token holders for the ERTH token. The contract address for the ERTH token on Moonbase Alpha is `0x08B40414525687731C23F430CEBb424b332b3d35`.
 
-将占位符替换为您的API密钥，然后在终端窗口运行以下指令。
+其中支持的端点之一是Token Holder端点，该端点将返回特定Token的所有Token持有者的列表。在本示例中，您可以检查ERTH的Token持有者。Moonbase Alpha上的ERTH Token合约地址为：`0x08B40414525687731C23F430CEBb424b332b3d35`。
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:207]:
```suggestion
Token Holder端点是Covalent支持的端点之一，该端点返回特定Token的所有Token持有者的列表。在本示例中，您可以检查ERTH的Token持有者。Moonbase Alpha上的ERTH Token合约地址为：`0x08B40414525687731C23F430CEBb424b332b3d35`。
```

File: builders/integrations/indexers/covalent.md [hunk 7]
```diff
@@ -176,11 +311,52 @@ def fetch_wallet_balance(address):
...
+!!! note 注意事项
+    The second parameter of `auth` is empty, because there is no password required - your API key is all that's needed.
+
+`auth`的第二个参数留空，因为不需要密码，只需要您的API密钥。
+
+## Increment {: #increment }
+
+[Increment](https://www.covalenthq.com/docs/increment/){target=_blank} is a no-code charting and reporting tool that enables users to build dynamic, personalized charts with data models. The tool directly encodes business logic—reach, retention, and revenue—into an SQL compiler that outputs valid SQL. Increment can convert any chart made with SQL and bake them into a standardized, open-sourced set of dimensions and measures, known as a model.
+
+[Increment](https://www.covalenthq.com/docs/increment/){target=_blank}是一种无代码图表和报告工具，使用户能够使用数据模型构建动态、个性化的图表。该工具直接将业务逻辑（包括覆盖范围、留存率和收入）编码到一个SQL编译器中，该编译器输出有效的SQL。Increment能够将使用SQL创建的任何图表转换并嵌入到一个标准化的、开源的维度和度量集中，被称为模型。
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:328]:
```suggestion
[Increment](https://www.covalenthq.com/docs/increment/){target=_blank}是一个无代码图表与报告生成工具，它让用户能够使用数据模型来构建动态、个性化的图表。该工具直接将业务逻辑（包括覆盖范围、留存率和收入）编码到一个SQL编译器中，然后由该编译器输出有效的SQL结果。Increment能够将使用SQL创建的任何图表转换并嵌入进一个标准化且开源的维度(dimension)和度量(measure)集合，其被称为模型。
```

File: builders/integrations/indexers/covalent.md [hunk 8]
```diff
@@ -176,11 +255,37 @@ def fetch_wallet_balance(address):
...
 ```
 
 !!! 注意事项
     `auth`的第二个参数留空，因为不需要密码，只需要您的API密钥。
 
+## Increment {: #increment }
+
+[Increment](https://www.covalenthq.com/docs/increment/){target=_blank}是一个无代码图表与报告生成工具，它让用户能够使用数据模型来构建动态、个性化的图表。该工具直接将业务逻辑（包括覆盖范围、留存率和收入）编码到一个SQL编译器中，然后由该编译器输出有效的SQL结果。Increment能够将使用SQL创建的任何图表转换并嵌入进一个标准化且开源的维度(dimension)和度量(measure)集合，其被称为模型。
+
+[![Example Increment chart](/images/builders/integrations/indexers/covalent/covalent-3.png)](https://www.covalenthq.com/platform/increment/#/?utm_source=moonbeam&utm_medium=partner-docs){target=_blank} *Click on the above image to try out Increment.*
```
Comment (eshaben) [inline builders/integrations/indexers/covalent.md:270]: @wuzhong-papermoon can you please provide a translation for:  *Click on the above image to try out Increment.*

**Comment by eshaben** [comment]
@wuzhong-papermoon can you please translate the one item I commented on and then approve the PR?

File: builders/integrations/indexers/covalent.md [hunk 9]
```diff
@@ -316,47 +260,32 @@ def fetch_wallet_balance(address):
...
+!!! 注意事项
+    `auth`的第二个参数留空，因为不需要密码，只需要您的API密钥。
 
 ## Increment {: #increment }
 
-[Increment](https://www.covalenthq.com/docs/increment/){target=_blank} is a no-code charting and reporting tool that enables users to build dynamic, personalized charts with data models. The tool directly encodes business logic—reach, retention, and revenue—into an SQL compiler that outputs valid SQL. Increment can convert any chart made with SQL and bake them into a standardized, open-sourced set of dimensions and measures, known as a model.
-
 [Increment](https://www.covalenthq.com/docs/increment/){target=_blank}是一个无代码图表与报告生成工具，它让用户能够使用数据模型来构建动态、个性化的图表。该工具直接将业务逻辑（包括覆盖范围、留存率和收入）编码到一个SQL编译器中，然后由该编译器输出有效的SQL结果。Increment能够将使用SQL创建的任何图表转换并嵌入进一个标准化且开源的维度(dimension)和度量(measure)集合，其被称为模型。
 
 [![Example Increment chart](/images/builders/integrations/indexers/covalent/covalent-3.png)](https://www.covalenthq.com/platform/increment/#/?utm_source=moonbeam&utm_medium=partner-docs){target=_blank} *Click on the above image to try out Increment.*
```
Comment (wuzhong-papermoon) [inline builders/integrations/indexers/covalent.md:270]:
```suggestion
[![Example Increment chart](/images/builders/integrations/indexers/covalent/covalent-3.png)](https://www.covalenthq.com/platform/increment/#/?utm_source=moonbeam&utm_medium=partner-docs){target=_blank} *点击上述图片尝试Increment。*
```

**Review by wuzhong-papermoon — APPROVED** [review]

**Pull request merged**